global.json

```diff
@@ -1,6 +1,6 @@
 {
 	"sdk": {
 		"version": "6.0.400",
-		"rollForward": "latestFeature"
+		"rollForward": "latestMajor"
```

Comment:
This is annoying if you have sdk 7 installed.. let me know if you think this should stay latestFeature

Reply:
Since backwards compatibility isn't generally an issue with .NET I think this is fine.